modules/rdg/src/userobjects/SlopeLimitingBase.C

```diff
@@ -52,14 +52,6 @@ SlopeLimitingBase::initialize()
   _lslope.clear();
 }
 
-void
-SlopeLimitingBase::threadJoin(const UserObject & y)
-{
-  const SlopeLimitingBase & pps = static_cast<const SlopeLimitingBase &>(y);
-
-  _lslope.insert(pps._lslope.begin(), pps._lslope.end());
```

Comment:
Yep! It is never called with GeneralUserObjects because we don't run them in threads. That also means that the data is never distributed so it never needs to be joined.